apps/mobile/src/features/Signers/Signers.container.tsx

```diff
@@ -1,9 +1,42 @@
-import { View, Text } from 'tamagui'
+import { View } from 'tamagui'
+import React, { useMemo } from 'react'
+
+import { SafeButton } from '@/src/components/SafeButton'
+
+import { SignersList } from './components/SignersList'
+import { Dimensions } from 'react-native'
+import { useSignersGroupService } from './hooks/useSignersGroupService'
+import { useRouter } from 'expo-router'
 
 export const SignersContainer = () => {
+  const { group, isFetching } = useSignersGroupService()
+  const router = useRouter()
+
+  const onImportSigner = () => {
+    router.push('/import-signers')
+  }
+
+  const signersSections = useMemo(() => {
+    if (!group.imported) {
+      return []
+    }
+
+    return group.imported?.data.length ? Object.values(group) : [group.notImported]
+  }, [group])
+
   return (
-    <View paddingHorizontal={'$3'} style={{ flex: 1 }}>
-      <Text>Signers</Text>
+    <View paddingTop="$6" gap="$6">
```

Comment:
I think that the padding makes it look weird. You don't expect the text to hide that early.
![grafik](https://github.com/user-attachments/assets/896c1906-b586-4257-8482-553e0af6c108)

Compare this to the pendingTx screen:
![grafik](https://github.com/user-attachments/assets/ac4cffe7-7782-41ed-8fb2-3f68e9fe4919)